Valet.podspec

```diff
@@ -1,15 +1,14 @@
 Pod::Spec.new do |s|
```

Comment:
can actually add `test_spec` DSL now to represent your tests by pods which means they would execute when publishing.

Can be in a follow up.

Reply:
Up to you @dfed I'd be ok with having it happen here.

Reply:
Well that'd be slick. I'm 💯 for this

Reply:
@dnkoutso mind if I grab you mañana?

Reply:
sure!

Reply:
Did this still need to happen?

Reply:
This work is here: https://github.com/square/Valet/pull/120